index.js

```diff
@@ -8,43 +8,65 @@ var hasFlag = function (flag) {
 	return pos !== -1 && (terminator !== -1 ? pos < terminator : true);
 };
 
-module.exports = (function () {
-	if ('FORCE_COLOR' in process.env) {
-		return true;
+var support = function (level) {
+	if (level <= 0) {
+		return null;
+	}
+	var result = {};
+	result.level = level;
+	result.has16 = result.hasBasic = true;
```

Comment:
Maybe we should only have `hasBasic`. As `has16` and `has16m` is IMHO a bit too close.